source/unified-test-format/unified-test-format.rst

```diff
@@ -754,12 +750,34 @@ See the `Stable API <../versioned-api/versioned-api.rst>`__ spec for more
 details on these fields.
 
 
-collectionData
-~~~~~~~~~~~~~~
+initialCollectionData
+~~~~~~~~~~~~~~~~~~~~~
 
-List of documents corresponding to the contents of a collection. This structure
-is used by both `initialData`_ and `test.outcome <test_outcome_>`_, which insert
-and read documents, respectively.
+List of documents corresponding to the contents of a collection. This
+structure is used by `initialData`_.
+
+The structure of this object is as follows:
+
+- ``collectionName``: Required string. See `commonOptions_collectionName`_.
+
+- ``databaseName``: Required string. See `commonOptions_databaseName`_.
+
+- ``collectionOptions``: Optional object. Options that MUST be passed to the
+  `create <https://docs.mongodb.com/manual/reference/command/create/>`_ command
+  when creating the collection. Test files SHOULD NOT specify ``writeConcern``
```

Comment:
Per my earlier comment in the JSON schema, I think you can enforce this (see: https://stackoverflow.com/a/30725034) and use `MUST NOT` here if that's preferable.

Reply:
Done.